src/test/kotlin/no/nav/medlemskap/common/TjenesteKallFeilmeldingerTest.kt

```diff
@@ -0,0 +1,299 @@
+import no.nav.medlemskap.ApplicationState
+import no.nav.medlemskap.clients.Services
+import no.nav.medlemskap.config.AzureAdOpenIdConfiguration
+import no.nav.medlemskap.config.Configuration
+import no.nav.medlemskap.createHttpServer
+import no.nav.medlemskap.domene.*
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.YearMonth
+
+class TjenesteKallFeilmeldingerTest {
+
+    protected fun RequestSpecification.When(): RequestSpecification {
+        return this.`when`()
+    }
+
+    protected inline fun <reified T> ResponseBodyExtractionOptions.to(): T {
+        return this.`as`(T::class.java)
+    }
+
+    companion object {
+
+        private val configuration = Configuration(
+                azureAd = Configuration.AzureAd(
+                        authorityEndpoint = "http://localhost/"
+                ))
+        private val services = Services(configuration)
+        private val openIdConfiguration = AzureAdOpenIdConfiguration("http://localhost", "", "", "")
+
+        private var applicationState = ApplicationState(false, false)
+
+        private lateinit var server: ApplicationEngine
+
+        @BeforeAll
+        @JvmStatic
+        fun startServer() {
+
+            if (!applicationState.running) {
+                val applicationServer = createHttpServer(
+                        applicationState = applicationState,
+                        useAuthentication = false,
+                        configuration = configuration,
+                        azureAdOpenIdConfiguration = openIdConfiguration,
+                        services = services,
+                        port = 7072,
+                        createDatagrunnlag = ::mockCreateDatagrunnlag
+                )
+
+                Runtime.getRuntime().addShutdownHook(Thread {
+                    applicationState.initialized = false
+                    applicationServer.stop(5000, 5000)
+                })
+
+                applicationServer.start()
+                applicationState.initialized = true
+                applicationState.running = true
+
+                RestAssured.baseURI = "http://localhost"
+                RestAssured.basePath = "/"
+                RestAssured.port = 7072
+                RestAssured.config = RestAssuredConfig.config().objectMapperConfig(ObjectMapperConfig.objectMapperConfig()
+                        .jackson2ObjectMapperFactory { _, _ -> objectMapper })
+            }
+        }
+
+    }
+
+    @Test
+    fun `Request body med fom som null får 400 respons`() {
+        val faktiskResponse = RestAssured.given()
+                .body(inputMedFomLikeNull)
+                .header(Header("Content-Type", "application/json"))
+                .post("/")
+                .then()
+                .statusCode(400)
+                .extract().asString()
+        println("Dette er responsen: $faktiskResponse")
```

Comment:
Disse er unødvendige, jeg antar at de bare henger igjen etter at du jobbet med å finne ut hvordan det fungerte..

Reply:
Du kan forøvrig gjerne gjøre asserts også på responsen!